R/download.R

```diff
@@ -135,9 +135,9 @@ dwnld.xwalk_zip_county <- function(pth = "https://www2.census.gov/geo/docs/maps-
 #' @description Downloads the crosswalk between zip codes and latitude longitude
 #' coordinates of the center for Texas
 #' @export
-dwnld.geo_zip <- function(state_fips) {
+dwnld.geo_zip <- function() {
 
-  tigris::zctas(state = state_fips) %>% 
+  tigris::zctas() %>% 
```

Comment:
@sadierichardson why do not subset to Texas in this ticket?

Reply:
@holmesjoli I'm not sure what changed, but it was giving an error "ZCTAs are only available by state for 2000 and 2010." so I removed the state code to make it run

Reply:
@sadierichardson gotcha I'll take a look!!